### ISSUE TITLE
false positive "no-unused-import": type used in mapping

### ISSUE THREAD

**Issue by 0xCourtney** [solhint-community/solhint-community#28]
solhint reports that `UD60x18` is not used.

```solidity
pragma solidity >=0.8.19;

import {UD60x18} from "@prb/math/UD60x18.sol";

contract SolhintTest {
    mapping(address user => UD60x18 amount) internal balance;
}
```

**Comment by juanpcapurro** [comment]
Thanks for reporting this and #29 

I have a PR up fixing both that should fix these two issues. 

Are you finding this issues in a public repo you could share? If that's the case, I'd like to use it to find any more false positives and I don't mind making a PR switching to solhint-community in the process, if you haven't done so already

**Comment by 0xCourtney** [comment]
> Are you finding this issues in a public repo you could share?

We're planning on making the repo public soon, I can't give too many details here though.

**Comment by juanpcapurro** [comment]
thanks for your report and your paitience @0xCourtney . This helped me learn a bit more about the AST and further simplify the rule's implementation. Should be deployed to 3.6.1  and 3.7.0-rc01 (I'm now using [release candidates](https://blog.capu.tech/updates-to-solhint-community-versioning.html) ) in 1-2 days
